### ISSUE TITLE
RandomShuffleQueue '_1_shuffle_batch/random_shuffle_queue' is closed and has insufficient elements (requested 10, current size 0)

### ISSUE THREAD

**Issue by AshwaqKhalil** [boncey/Flickr4Java#233]
Hello
I tried to read SVHN dataset as png in tensorflow. I used convolution neural network, and TFReacords to read the data. Also, I stored the labels in text file to read them in the code.
during the reading images I change their size to [32,32]

this link shows the code:: [code](https://www.dropbox.com/s/skx0jml4y02lba3/SVHN_v2.py?dl=0)

I have two problems which give me the error::

the first one is shown in bellow statement (which it is the reason of the second problem)

`image = tf.reshape(image, [IMAGE_SIZE, IMAGE_SIZE, DEPTH])`
the second problem is ::

> RandomShuffleQueue '_1_shuffle_batch/random_shuffle_queue' is closed and has insufficient elements (requested 10, current size 0)

The full tracback of the error is [here](https://www.dropbox.com/s/qvt4jwk69quw6u8/TF_error.txt?dl=0)

I asked this question in stackoverflow  too: [the question](https://stackoverflow.com/questions/45088946/randomshufflequeue-1-shuffle-batch-random-shuffle-queue-is-closed-and-has-ins)

* Tensorflow
